### DISCUSSION TITLE
Make LED(s) flash from web interface / via command

### DISCUSSION THREAD

**Discussion by darkmattercoder** [arendst/Tasmota#14767] [Q&A] [answered]
Rationale:

In industrial applications, it is pretty common that devices on any common bus or network have the ability to flash an LED for identification purposes.

That way one can set any device to flash and is able to find it physically to do stuff with it.

Request:

I would like to see the same for tasmota devices. Imagine you have a network containing multiple devices which you can access via ip but do not know where they physically reside. Or/and you have as well some devices in the network which still open accesspoints and you do also not know which ap corresponds to a given device. 

So I suggest introducing a possibility to make LEDs  flash with a single command or a button in the UI.

What do you think?

**Comment by sfromis** [comment, marked as answer]
If you config the led as a relay, you can use the command `Power blink` to, well, blink - after possibly having configured `BlinkCount` and `BlinkTime`. If you config it as a PWM pin, you also get variable brightness.

**Reply by darkmattercoder** [reply]
While this sounds like a possible solution, it requires changing configuration. The requested "blink mode" could do exactly that, but preconfigured, oneshot

**Reply by sfromis** [reply]
If you restart a device with standard config, it should already blink for a short time while getting Wifi and MQTT connections.
